core/application/impl/app_configuration_impl.cpp

```diff
@@ -324,10 +342,35 @@ namespace kagome::application {
     if (vm.end() != vm.find("unix_slots")) is_unix_slots_strategy_ = true;
 
     find_argument<std::string>(
-        vm, "genesis", [&](std::string const &val) { genesis_path_ = val; });
+        vm, "genesis", [&](const std::string &val) { genesis_path_ = val; });
 
     find_argument<std::string>(
-        vm, "base_path", [&](std::string const &val) { base_path_ = val; });
+        vm, "base_path", [&](const std::string &val) { base_path_ = val; });
+
+    std::vector<std::string> boot_nodes;
+    find_argument<std::vector<std::string>>(
+        vm, "bootnodes", [&](const std::vector<std::string> &val) {
+          boot_nodes = val;
+        });
+    boot_nodes_.reserve(boot_nodes.size());
+    for (auto &addr_str : boot_nodes) {
+      auto ma_res = libp2p::multi::Multiaddress::create(addr_str);
+      if (not ma_res.has_value()) {
+        auto err_msg = "Bootnode '" + addr_str
+                       + "' is invalid: " + ma_res.error().message();
+        logger_->error(err_msg);
+        std::cout << err_msg << std::endl;
```

Comment:
I'm terribly sorry, it seems like I missed some conversation, but why do we need this cout? And, since it is not asked the first time, maybe it would make sense to comment this line.

Reply:
Fails of CLI arguments processing should be shown immediately, to fix it instantly. Log can be writing to file and we should be do additional steps to find reason of problem.